server/jetstream_benchmark_test.go

```diff
@@ -1174,3 +1174,161 @@ func BenchmarkJetStreamKV(b *testing.B) {
 		)
 	}
 }
+
+func BenchmarkJetStreamObjStore(b *testing.B) {
+	const (
+		verbose            = false
+		objStoreNamePrefix = "B_"
+		keyPrefix          = "K_"
+		seed               = 12345
+		initKeys           = true
+	)
+
+	type ObjectSizeRange struct {
+		minBytes int
+		maxBytes int
+	}
+	// benchmarkOptions matrix
+	benchmarkOptions := struct {
+		replicaAndClusterSize []int
+		numKeys               []int
+		objectSizeRange       [][]int
+		rwRatio               []float64
+		storageType           []nats.StorageType
+	}{
+		[]int{1, 3},             // 1 replica, 3 replicas
+		[]int{10, 1000, 100000}, // 10 keys, 1K keys, 100K keys
+		[][]int{
+			{1024, 1024}, // 1KB byte objects
+			//{102400, 102400}, // 1MB byte objects
+		},
+		[]float64{1.0, 0.0, 0.8},                                 // readOnly, writeOnly, 80% read
+		[]nats.StorageType{nats.MemoryStorage, nats.FileStorage}, // memory storage, file storage
+	}
+
+	// run benchmarks using options matrix
+	for _, clusterSize := range benchmarkOptions.replicaAndClusterSize {
+		replicas := clusterSize
+		for _, numKeys := range benchmarkOptions.numKeys {
+			for _, objectSizeRange := range benchmarkOptions.objectSizeRange {
+				minBytes, maxBytes := objectSizeRange[0], objectSizeRange[1]
+				for _, rwRatio := range benchmarkOptions.rwRatio {
+					var rwRatioType string
+					switch rwRatio {
+					case 1.0:
+						rwRatioType = "readOnly"
+					case 0.0:
+						rwRatioType = "writeOnly"
+					default:
+						rwRatioType = fmt.Sprintf("rwRatio=%0.2f%%", rwRatio)
+					}
+					for _, storageType := range benchmarkOptions.storageType {
+						bName := fmt.Sprintf(
+							"N=%d,R=%d,K=%d/storageType=%s/minBytes=%db,maxBytes=%db/%s",
```

Comment:
Don't add slashes in the test name, it's a special character used by benchmark to define hierarchical sub-tests.
In this case, you're not really doing a hierarchy, so just use commas

Also `minBytes` and `maxBytes` are ambiguous. How about `minObjSize`, `maxObjSize`